src/sentry/models/installation.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+
+from django.db import models, IntegrityError, transaction
+
+from sentry.db.models import Model, FlexibleForeignKey
+
+
+class OrganizationInstallation(Model):
+    __core__ = False
+
+    organization = FlexibleForeignKey('sentry.Organization')
+    installation = FlexibleForeignKey('sentry.Installation')
+
+    class Meta:
+        app_label = 'sentry'
+        db_table = 'sentry_organizationinstallation'
+        unique_together = (('organization', 'installation'),)
+
+
+class Installation(Model):
+    __core__ = False
+
+    organizations = models.ManyToManyField('sentry.Organization',
+                                           related_name='installations',
+                                           through=OrganizationInstallation)
+    provider = models.CharField(max_length=64)
+    installation_id = models.CharField(max_length=64)
+    external_organization = models.CharField(max_length=64, null=True)
```

Comment:
this was my way of trying to make it flexible for things other than github, but you're right and i should probably just assume these are always going to be necessary